### ISSUE TITLE
Is there a python version for getting the labels?

### ISSUE THREAD

**Issue by sandeepnmenon** [mohamedadaly/caltechcv-image-labeler#3]
I am trying to use the Caltech lanes dataset to train my model written in Pytorch. Wanted to adopt the same code to read the labels in my dataloader.
Are there any suggestions?

**Comment by sandeepnmenon** [comment]
Reading the labels using scipy loadmat.
Wrote a visualizer code which can be found here https://github.com/eeyrw/LaneDatasetInspector/blob/master/CaltechLanesDbInspector.py
